src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -2960,29 +2929,6 @@ public static void InvokeQuotedLambda(bool useInterpreter)
             Assert.Equal(6, d());
         }
 
-        [Theory(Skip = "870811")]
```

Comment:
There is an effort to hook up AOT/.net native test runs to the CI. I do not know how close that to being done. However tests may already be run as a part of the .net native builds, so additional failure could cause problems.

It seems safer to keep the test disabled.